webpack.config.js

```diff
@@ -41,6 +41,10 @@ module.exports = {
         loader: 'babel-loader'
       },
       {
+        test: /\.(jpe?g|png|gif|svg)$/i,
```

Comment:
You do not appear to be using `file-loader` and thus should not need this config.